t/admin/routes3.t

```diff
@@ -700,3 +700,103 @@ passed
     }
 --- response_body
 passed
+
+
+
+=== TEST 20: set route(id: 1, parameters with boolean values)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                      "uri": "/index.html",
+                      "enable_websocket": true,
+                      "upstream": {
+                          "type": "roundrobin",
+                          "nodes": {
+                              "127.0.0.1:8080":1
+                          }
+                      }
+                  }]],
+                [[{
+                    "node": {
```

Comment:
This test is only for setting up route 1, we don't need to check the raw response.

Reply:
update